x/retry/observer.go

```diff
@@ -242,9 +242,5 @@ func (c call) maxAttemptsError(err error) {
 }
 
 func getErrorName(err error) string {
-	errCode := yarpcerrors.ErrorCode(err)
-	if errCode == yarpcerrors.CodeOK {
-		return "unknown_internal_yarpc"
```

Comment:
can we keep this around? We should make sure we can distinguish between properly wrapped errors and "unwrapped" errors

Reply:
Done